src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/rectangle.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, 0.6);
+$mainColor: $white;
+$boxShadow: 0 0 20px 0 rgba($black, 0.3);
+$borderSize: 2px;
+
+.rectangle {
+    position: absolute;
+    background: transparent;
+    border: $borderSize solid $mainColor;
+    z-index: 1000;
```

Comment:
I'd like to try keep the `z-index` values lower this time by correctly using [stacking contexts](https://getpocket.com/a/read/282540626). So please try using an as low level as possible (maybe `1` is already enough).